va/http.go

```diff
@@ -334,7 +334,11 @@ func (va *ValidationAuthorityImpl) extractRequestTarget(req *http.Request) (iden
 
 	reqIP, err := netip.ParseAddr(reqHost)
 	if err == nil {
-		if va.isReservedIPFunc(reqIP.AsSlice()) {
+		reserved, _, err := va.isReservedIPFunc(reqIP.AsSlice())
```

Comment:
Here we convert from a netip.Addr to a net.IP, and then we immediately convert back inside IsReservedIP. Can we skip both those conversions by having IsReservedIP take a netip.Addr directly?